templates/statistics/numbers.html

```diff
@@ -2,6 +2,9 @@
 {% extends "layout.html" %}
 {% set title = _("Numbers") %}
 {% block text %}
+<script>
+{{ states.label_js() }}
+</script>
```

Comment:
Inline JS? :/

Reply:
Otherwise I would need the templated numbers.js again that I just removed.
Definitively not the best but easiest to ensure different entrypoinst for smaller bundles.

Reply:
Hmm, the only purpose of this is to get the metadata of the different states into JS somehow. That whole `templates/macros/states.html` bothers me a lot anyway with the same stuff defined three times here and then in `templates/modals/base.html` as well.

If I remember correctly, we only really need that for the statistics to get the colors and label texts for the pie charts. However, we have all the different badges available in the table between the charts, so why not put that into their data attributes and parse it from there? So the only stuff that needs to be fed in dynamically would be the numbers and those could be parsed from the table as well (or pulled via the API)...

It is also used in the list to get the HTML for the different state labels in the table, but we would also have that already available from the modal by doing something like `$('.badge.OVERFLOW').clone()` or so.